.dev.env

```diff
@@ -0,0 +1,9 @@
+MYSQL_ROOT_PASSWORD=password
+MYSQL_DATABASE=rocketdb
+MYSQL_USER=rocketdb
+MYSQL_PASSWORD=rocketdb
+
+THERAIDMAPPER_DBIP=mariadb
```

Comment:
Guess we could just use the configuration above for user and password?

Reply:
Each container is expecting different variable names which we have them defined twice. The DBIP is a bit different since that is the name of the MariaDB container that is used for DNS resolution